Examples/minimum_user_for_s2e/src/src_user/Drivers/Com/gs.c

```diff
@@ -145,14 +158,14 @@ static void GS_load_default_driver_super_init_settings_(DriverSuper* p_super)
     DSSC_set_rx_frame_size(p_stream_config, GS_RX_FRAME_SIZE); // 可変長
```

Comment:
[IMO] これ結構ミスリーディングだし
```suggestion
    DSSC_set_rx_frame_size(p_stream_config, -1); // 可変長
```

として， `GS_RX_FRAME_SIZE` を消していい気がする？